test/server/graphql/v2/mutation/ExpenseMutations-legacy.test.js.snap

```diff
@@ -23,7 +23,9 @@ exports[`server/graphql/v2/mutation/ExpenseMutations processExpense PAY Multi-cu
 | type   | amount | netAmountInCollectiveCurrency | paymentFee | currency | hostCurrency | hostCurrencyFxRate | To    | From  | Host      | isRefund |
 | ------ | ------ | ----------------------------- | ---------- | -------- | ------------ | ------------------ | ----- | ----- | --------- | -------- |
 | CREDIT | 1700   | 1600                          | -110       | USD      | NZD          | 1.1                | Payee | Babel | PayeeHost | false    |
-| DEBIT  | -1600  | -1700                         | -100       | USD      | USD          | 1                  | Babel | Payee | OSC       | false    |"
+| DEBIT  | -1600  | -1700                         | -100       | USD      | USD          | 1                  | Babel | Payee | OSC       | false    |
+| CREDIT | 160    | 160                           | 0          | USD      | NZD          | 1.1                | PayeeHost | Payee     | PayeeHost | false    |
+| DEBIT  | -160   | -160                          | 0          | USD      | NZD          | 1.1                | Payee     | PayeeHost | PayeeHost | false    |"
```

Comment:
Had to update the snapshots because of the link we now make in https://github.com/opencollective/opencollective-api/pull/10194/files#diff-517b6b387d3fb789d7bda4d342aa30454e69c46c68db511a7d342acf7b8e30e0R704, the tests look for transactions by expense ID and therefore now return the host fees.